.github/workflows/go.yml

```diff
@@ -22,18 +22,29 @@ jobs:
 
   tests:
     name: Tests
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go_versions: [ '1.20', '1.21', '1.22' ]
+        go: [ '1.20', '1.21', '1.22' ]
+        os: [ubuntu-latest, windows-2022, macos-14]
```

Comment:
why some are the latest while the others are fixed?

Reply:
IIRC `latest` there is not what we have here.